bundles/ranvier-combat/commands/kill.js

```diff
@@ -16,26 +17,35 @@ module.exports = (srcPath) => {
         return Broadcast.sayAt(player, "You're too busy fighting!");
       }
 
-      const npc = Parser.parseDot(args, player.room.npcs);
-      if (!npc) {
+      const possibleTargets = [
+        ...player.room.npcs,
+        ...player.room.players
+      ];
+
+      const target = Parser.parseDot(args, possibleTargets);
+      if (!target) {
         return Broadcast.sayAt(player, "They aren't here.");
       }
 
-      if (npc.hasBehavior('pacifist')) {
-        return Broadcast.sayAt(player, `${npc.name} is a pacifist and will not fight you.`);
+      if (target.hasBehavior && target.hasBehavior('pacifist')) {
```

Comment:
I'd prefer this checked `target instanceof Npc` instead of checking for existence of hasBehavior. If you want to be _really_ cool you could make this easier for everyone and add a `isNpc()` method to the base `Character` class which returns false by default but `Npc` overrides to return true.